casatables/src/lib.rs

```diff
@@ -529,6 +529,50 @@ where
     )
 }
 
+// TableDesc
+
+pub struct TableDesc {
+    handle: *mut glue::GlueTableDesc,
+    exc_info: glue::ExcInfo,
+}
+
+impl TableDesc {
+    pub fn new(stype: &str) -> Self {
```

Comment:
It would be nice to add rustdoc comments to the public API functions to explain them, although I haven't done a great job of this myself. Especially with this "type" string whose purpose seems unclear. (If we can't figure what exactly the point of the parameter is, we can at least document that it gets saved in such-and-such a field of the table, or something.)